ARCHITECTURE.md

```diff
@@ -7,10 +7,9 @@ This document aims to answer the question *Where is the code that does X?*
 The directory structure is broken down as follows:
 
 - `cmd/` - These CLI tools are primarily used during development for end-to-end testing without needing to pull the library into trivy/tfsec etc.
-- `checks` - All of the checks are defined in this directory.
+- `checks` - All the checks are defined in this directory.
 - `commands` - All Node-collector commands are defined in this directory.
-- `pkg/spec` - Logic to handle standardized specs such as CIS.
-- `pkg/rules` - This package exposes internal rules, and imports them accordingly (see _rules.go_).
+- `pkg/rules` - This package exposes internal checks, and imports them accordingly (see _rules.go_).
```

Comment:
I don't understand what this means. also 1. didn't we rename rules to checks? 2. shouldn't this code be in trivy?

Reply:
It's no longer used. I'll remove it.